src/decode.rs

```diff
@@ -85,7 +87,7 @@ impl<'a, I: AsRef<[u8]>> DecodeBuilder<'a, I> {
     /// Setup decoder for the given string using the given alphabet.
     /// Preferably use [`bs58::decode`](../fn.decode.html) instead of this
     /// directly.
-    pub fn new(input: I, alpha: &'a [u8; 58]) -> DecodeBuilder<'a, I> {
+    pub fn new(input: I, alpha: AlphabetCow<'a>) -> DecodeBuilder<'a, I> {
```

Comment:
I'd prefer to avoid breaking changes yet. I think you can leave `new` as it was (and I'll likely deprecate it soon) and add a new `pub(crate) from_input(input: I) -> DecodeBuilder<'static, I>` for `fn decode` which uses a borrow on `Alphabet::DEFAULT`.